optuna_dashboard/ts/components/TrialTable.tsx

```diff
@@ -293,6 +332,41 @@ export const TrialTable: FC<{
             />
           </Box>
         </Grid>
+        <Grid item xs={12}>
+          <Box margin={1}>
+            <Typography variant="h6" gutterBottom component="div">
+              Trial tell operations
+            </Typography>
+            <form onSubmit={handleSubmit}>
+              <Box margin={1}>
+                {objectiveValues.map((value, i) => (
+                  <TextField
+                    id={`objective-${i}`}
+                    key={`objective-${i}`}
+                    label={`Objective ${i}`}
+                    type="number"
+                    value={objectiveValues[i]}
+                    onChange={(e: ChangeEvent<HTMLInputElement>) =>
+                      handleChangeValue(i, e)
+                    }
+                  />
+                ))}
+              </Box>
+              <Box margin={1}>
+                <Button variant="contained" type="submit">
+                  Submit
+                </Button>
+                <Button
+                  variant="outlined"
+                  color="error"
+                  onClick={handleFailTrial}
+                >
+                  Fail trial
+                </Button>
+              </Box>
+            </form>
+          </Box>
+        </Grid>
```

Comment:
Sorry I'm not very familiar with react but will this component always be shown even for completed trials? It'd be nice if we could hide the component for finished trials. What do you think?

Reply:
That sounds reasonable to me +1

Reply:
Fixed.

Reply:
Seemed good, let me resolve this comment. Thanks 🙇